### PR TITLE
build: test / wheels in python 3.13

### PR DESCRIPTION
**Pull request by lgray**
I'm not quite sure how to add freethreaded python here though. Some environment variable right @henryiii?

### CODE REVIEW

**Comment by henryiii** [comment]
https://github.com/milesgranger/cramjam/issues/165 is required for Python 3.23 support.

I don't know if PyO3 supports free-threaded yet. I don't think GitHub Actions does yet either, you need deadsnakes or cibuildwheel, something like that.

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -28,6 +28,7 @@ jobs:
           - '3.10'
           - '3.11'
           - '3.12'
+          - '3.13-dev'
```
Comment (henryiii) [inline .github/workflows/CI.yml:31]: You should use '3.13' and set allow-prereleases: true instead. That won't need to be updated in the future.

**Comment by lgray** [comment]
I'm going to close this PR since it's obviously a lot more work. Wanted to see if there were some easy wins. :-)

Hopefully we get 3.13t in actions soon!

**Comment by milesgranger** [comment]
@lgray [2.8.4rc3](https://pypi.org/project/cramjam/2.8.4rc3/)

**Pull request closed without merging**